core/store/src/cold_storage.rs

```diff
@@ -232,6 +277,21 @@ impl StoreWithCache<'_> {
     ) -> io::Result<T> {
         option_to_not_found(self.get_ser(column, key), format_args!("{:?}: {:?}", column, key))
     }
+
+    pub fn insert_state_to_cache_from_op(
+        &mut self,
+        op: &TrieRefcountChange,
+        shard_uid_key: &StoreKey,
+    ) {
+        debug_assert_eq!(
+            DBCol::State.key_type(),
+            &[DBKeyType::ShardUId, DBKeyType::TrieNodeOrValueHash]
+        );
+        self.cache.insert(
+            (DBCol::State, join_two_keys(shard_uid_key, &op.hash().as_ref().to_vec())),
```

Comment:
```suggestion
            (DBCol::State, join_two_keys(shard_uid_key, op.hash().as_bytes())),
```
No need to create a vector.